### ISSUE TITLE
Can't install it on FreeBSD

### ISSUE THREAD

**Issue by tsgan** [sandfoxme/journald-native#4]
I can't install it on FreeBSD 11.1-RELEASE-p4:
```
# gem install journald-native -v '1.0.10' --source 'https://rubygems.org/'
Building native extensions. This could take a while...
ERROR:  Error installing journald-native:
	ERROR: Failed to build gem native extension.

    current directory: /usr/local/lib/ruby/gems/2.4/gems/journald-native-1.0.10/ext/journald_native
/usr/local/bin/ruby24 -r ./siteconf20180810-17968-eivwpi.rb extconf.rb
checking for systemd/sd-journal.h... no
checking for -lsystemd... no
checking for sd_journal_print()... no
checking for -lsystemd-journal... no
checking for sd_journal_print()... no
creating extconf.h
creating Makefile

current directory: /usr/local/lib/ruby/gems/2.4/gems/journald-native-1.0.10/ext/journald_native
make "DESTDIR=" clean

current directory: /usr/local/lib/ruby/gems/2.4/gems/journald-native-1.0.10/ext/journald_native
make "DESTDIR="
compiling journald_native.c
In file included from journald_native.c:21:
./sd_journal.h:60:10: warning: Compiling dummy version of the gem for non-Linux OS [-W#warnings]
        #warning Compiling dummy version of the gem for non-Linux OS
         ^
In file included from journald_native.c:21:
In file included from ./sd_journal.h:62:
./sd_journal_dummy.h:35:8: error: redefinition of 'iovec'
struct iovec {
       ^
/usr/include/sys/_iovec.h:43:8: note: previous definition is here
struct iovec {
       ^
1 warning and 1 error generated.
*** Error code 1

Stop.
make: stopped in /usr/local/lib/ruby/gems/2.4/gems/journald-native-1.0.10/ext/journald_native

make failed, exit code 1

Gem files will remain installed in /usr/local/lib/ruby/gems/2.4/gems/journald-native-1.0.10 for inspection.
Results logged to /usr/local/lib/ruby/gems/2.4/extensions/amd64-freebsd-11/2.4/journald-native-1.0.10/gem_make.out```

**Comment by sandfoxme** [comment]
Just when I thought it was too easy :D

@tsgan first question - why do you need this gem on FreeBSD? As I know, systemd is a linux-only system. Dummy version was created for some colleagues who use Mac and Windows as development machines

Also, do you have any ideas how to fix this?

**Comment by sandfoxme** [comment]
Fixed in 1.0.11. Also fixed crash on call to print() in dummy mode on FreeBSD. Thanks for reporting!
